packages/client/mutations/AddCommentMutation.ts

```diff
@@ -63,6 +69,8 @@ const AddCommentMutation: StandardMutation<TAddCommentMutation> = (
     variables,
     updater: (store) => {
       const payload = store.getRootField('addComment')
+      const {meetingId} = variables
+      payload.setValue(meetingId, 'meetingId')
```

Comment:
this pattern is a little too dangerous. Remember, the updater function is shared by mutations & subscriptions because we guarantee that they'll have the same payload. if we break that contract, then subscriptions can break.

if you need a meetingId, it's best for the server to just return it

Reply:
Thanks, that's helpful to know. I've added the `meetingId` to the payload